autoload/airline/extensions/lsp.vim

```diff
@@ -0,0 +1,63 @@
+" MIT License. Copyright (c) 2013-2019 François-Xavier Carton et al.
+" Plugin: https://github.com/prabirshrestha/vim-lsp
+" vim: et ts=2 sts=2 sw=2
+
+scriptencoding utf-8
+
+function! s:airline_lsp_count(cnt, symbol)
```

Comment:
please add the `abort` attribute (as mentioned in the github actions linter. (also see the other function definitions).

Reply:
I did not change the ones in ale.vim, from which I copy-pasted lsp.vim.